### ISSUE TITLE
Rule AA0137 (Unused variable) not working for global variables defined on the same line separated with comma

### ISSUE THREAD

**Issue by rvanbekkum** [microsoft/AL#6385]
**Describe the bug**
Rule AA0137 "Variable * is unused in *" does not work for global variables in an object that are defined on the same line separated by a comma (i.e., `GlobalUnusedVariable, GlobalAnotherUnusedVariable : Boolean;`). For local variables this works fine.

**To Reproduce**
Example (see screenshot):
```al
pageextension 50100 CustomerListExt extends "Customer List"
{
    trigger OnOpenPage();
    var
        UnusedVariable: Boolean;
        AnotherUnusedVariable, YetAnotherUnusedVariable : Boolean;
    begin
        Message('App published: Hello world');
    end;

    var
        GlobalUnusedVariable: Boolean;
        GlobalAnotherUnusedVariable, GlobalYetAnotherUnusedVariable : Boolean;
}
```

**Expected behavior**
Warnings for unused global variables that are defined like this.

**Screenshots**
![AA0137](https://user-images.githubusercontent.com/7383241/102608854-d735ef00-412a-11eb-9b39-4e73cc5472a1.png)

**5. Versions:**

 - AL Language: 6.1.381323

 - Business Central: Version: W1 17.2 (Platform 17.0.19353.19730 + Application 17.2.19367.19735)

**Comment by fvet** [comment]
In the latest AL version 9.0.615906, it shows `AA0205 `instead ... ?

![image](https://user-images.githubusercontent.com/25268332/168068452-0e3dbf4e-0489-4730-89c6-772b7114d56d.png)

**Comment by mazhelez** [comment]
Thanks for reporting this issue. Sorry we haven’t completed it yet, but we’ve had to prioritize elsewhere. We’re planning to give the CodeCop engine and its rules an overhaul in a future major release. Thanks for your patience.

Going forward we will not track progress on CodeCop issues here on GitHub. Hence, we are closing this issue.

**Comment by pri-kise** [comment]
Still not fixed for RunTime 11.0.x (Just for Reference, if someone searches for this issue in the future)
